pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -1305,23 +1305,25 @@ func registerCDC(r registry.Registry) {
 	// the first account on the assume-role chain:
 	// cdc-roachtest-intermediate@cockroach-ephemeral.iam.gserviceaccount.com. See
 	// https://cloud.google.com/iam/docs/create-short-lived-credentials-direct.
-	//
-	// TODO(rui): Change to a shorter test as it just needs to validate
-	// permissions and shouldn't need to run a full 30m workload.
 	r.Add(registry.TestSpec{
 		Name:             "cdc/pubsub-sink/assume-role",
 		Owner:            `cdc`,
 		Benchmark:        true,
 		Cluster:          r.MakeClusterSpec(4, spec.CPU(16)),
 		Leases:           registry.MetamorphicLeases,
-		CompatibleClouds: registry.AllExceptAWS,
+		CompatibleClouds: registry.OnlyGCE,
 		Suites:           registry.Suites(registry.Nightly),
 		RequiresLicense:  true,
 		Run: func(ctx context.Context, t test.Test, c cluster.Cluster) {
 			ct := newCDCTester(ctx, t, c)
 			defer ct.Close()
 
-			ct.runTPCCWorkload(tpccArgs{warehouses: 1, duration: "30m"})
+			// Bump memory limit (too low in 22.2)
+			if _, err := ct.DB().Exec("SET CLUSTER SETTING changefeed.memory.per_changefeed_limit = '512MiB';"); err != nil {
```

Comment:
From what I can tell, [this is the default on `master` already](https://github.com/cockroachdb/cockroach/blob/ba6423f916dc4006bb590c786cf53ad2b0213417/pkg/ccl/changefeedccl/changefeedbase/settings.go#L46-L51)? Instead, shall we just change this on `release-22.2`?

Reply:
Didn't want to make direct 22.2 patch and wanted to backport; do you think it might be okay to merge this; backport it, and then remove this cluster setting since it's the default?

Reply:
Sg.